app/templates/adf-cli-acs-aps-template/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM nginx
+
+COPY nginx.conf /etc/nginx/nginx.conf
+
+WORKDIR /usr/share/nginx/html
+COPY demo-shell/dist/ .
```

Comment:
I think there will be only the `dist` folder in the generated app, no demo-shell folder.

Reply:
right